exercise/static/exercise/chapter.js

```diff
@@ -571,9 +579,8 @@
 					//exercise.hideLoader();
 					var input = $(data);
 					if (exercise.quiz) {
-						var badge = input.find('.badge').eq(2).clone();
```

Comment:
Do you have idea what this `.badge` refers to? Is it just points/status or is it some achievement thing?

Reply:
It was the badge on the modal for the questionnaire _(the 0/20 on the picture)_. It was given to the `modalSuccess` there.

![image](https://user-images.githubusercontent.com/43036333/90761992-edeb4700-e2ec-11ea-9943-dc2d09073873.png)